languages/en_US.po

```diff
@@ -674,644 +837,780 @@ msgstr ""
+#: admin_style.php:557
+msgid "页脚样式"
+msgstr "Footer styles"
 
-#: admin_style.php:458
-msgid "评论表情"
-msgstr "Comment Emoji"
+#: admin_style.php:561
+msgid "传统"
+msgstr "Traditional"
 
-#: admin_style.php:464
+#: admin_style.php:565
 msgid ""
-"开启后，评论时可输入表情。开启后仍需安装表情插件才可正常使用。目前仅支持来自 "
-"mayuko 的 Alu 表情插件，请前往 <a href=\"https://doc.flyhigher.top/mdx/zh-CN/"
-"config/emoji-in-comment/\" target=\"_blank\">MDx文档</a> 下载插件安装包。"
+"在此设定页脚样式。<strong>如果选择“简单”样式，那么页脚格言将不会显示。</"
+"strong>"
 msgstr ""
-"Enable support for emojis in comments. Requires extra plugin (Alu emojis by "
-"mayuko). Please visit <a href=\"https://doc.flyhigher.top/mdx/zh-CN/config/"
-"emoji-in-comment/\" target=\"_blank\">MDx documentation</a> to download the "
-"plugin."
+"Set footer styles here. <strong> Footer quotes will not be displayed for "
+"\"simple\" style.</strong>"
 
-#: admin_style.php:469, functions.php:769
-msgid "文末信息"
-msgstr "End-of-post message"
+#: admin_style.php:569
+msgid "页脚格言"
+msgstr "Footer quotes"
 
-#: admin_style.php:471
-msgid ""
-"在这里编辑文末信息。文末信息会显示在每篇文章的底部，留空则不会显示。支持 "
-"<code>HTML</code> 格式。<code>--PostLink--</code> 会被替换为链接到当前文章的"
-"文章标题，<code>--PostURL--</code> 会被替换为链接到当前文章的当前文章 URL（大"
-"小写敏感）。"
-msgstr ""
-"The message shown at the end of each post. Leave it blank to disable this "
-"feature. <code>HTML</code> codes are supported. <code>--PostLink--</code> "
-"will be replaced with post title linked to the post; <code>--PostURL--</"
-"code> will be replaced with post URL linked to this page."
+#: admin_style.php:571
+msgid "使用一言 API（常规）"
+msgstr "Use Hitokoto.cn API (General)"
 
-#: admin_style.php:475
-msgid "顶部栏显示内容"
-msgstr "Title bar content"
+#: admin_style.php:572
+msgid "使用一言 API（古诗词）"
+msgstr "Use Hitokoto.cn API (Chinese Poetry)"
 
-#: admin_style.php:479
-msgid "博客名称"
-msgstr "Blog name"
+#: admin_style.php:573
+msgid "使用一言 API（自定义）"
+msgstr "Use Hitokoto.cn API (custom)"
 
-#: admin_style.php:480
-msgid "自定义 Logo"
-msgstr "Custom Logo"
+#: admin_style.php:574
+msgid ""
+"这句话会显示在每个页面的页脚，如果不希望显示，请留空。若调用一言 API，则每次"
+"页面刷新后都会显示不同的格言。此 API 来自第三方，请注意安全风险。"
+msgstr ""
+"This quote will be shown at the footer of each page. Leave it blank to "
+"disable it. For Hitokoto.cn APIs, you will receive a different quote per "
+"refreshing. This API is from a third-party, please be aware of the security "
+"risks."
 
-#: admin_style.php:481
-msgid "自定义名称"
-msgstr "Custom name"
+#: admin_style.php:575
+msgid ""
+"如使用自定义 API，你需要在括号内填写 API URL 并确保 API 返回 <code>{\"text\":"
+"\"句子\"}</code> 的 JSON 格式。<a href=\"https://doc.flyhigher.top/mdx/zh-CN/"
+"config/custom_api/\" target=\"_blank\">详细信息</a></code>"
+msgstr ""
+"If you are using a custom API, you'll need to fill-in the API URL in "
+"parentheses and make sure that the API returns the JSON format <code>{\"text"
+"\":\"quote\"}</code> <a href=\"https://doc.flyhigher.top/mdx/en-US/config/"
+"custom_api/\" target=\"_blank\">Learn More</a></code>"
 
```

Comment:
The doc seems to be unavailable (for both `zh-CN` and `en-US` versions)
https://doc.flyhigher.top/mdx/zh-CN/config/custom_api/

Reply:
The English doc is not fully translated

Reply:
Since I'm still testing the theme, the doc has not been updated yet